base/applications/taskmgr/procpage.c

```diff
@@ -400,6 +400,10 @@ void ProcessPageShowContextMenu(DWORD dwProcessId)
     DWORD        dwDebuggerSize;
     HKEY         hKey;
 
+    if (dwProcessId == 0)
+    {
+        return;
+    }
```

Comment:
Preserving existing code style:
```suggestion
    if (dwProcessId == 0)
        return;

```